metrics/web/test/features/auth/test_utils/signed_in_auth_store_mock.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:metrics/features/auth/presentation/state/auth_store.dart';
+import 'package:mockito/mockito.dart';
+import 'package:rxdart/rxdart.dart';
+
+/// Mock implementation of the [AuthStore] ensures that a user is already logged in into the app.
+class SignedInAuthStoreMock extends Mock implements AuthStore {
+  final BehaviorSubject<bool> _isLoggedInSubject = BehaviorSubject();
+
+  @override
+  bool get isLoggedIn => _isLoggedInSubject.value;
+
+  SignedInAuthStoreMock() {
+    when(subscribeToAuthenticationUpdates())
+        .thenAnswer((_) => _isLoggedInSubject.add(true));
+    when(loggedInStream).thenAnswer((_) => _isLoggedInSubject.stream);
+    when(signOut()).thenAnswer((_) => _isLoggedInSubject.add(false));
```

Comment:
It seems we should stub method implementations directly in tests according to the [Mockito best practices](https://pub.dev/packages/mockito#best-practices).